### PR TITLE
Update BestpracticesTemplate.md

### PR DESCRIPTION
**Pull request by jlovejoy**
added language to specify inbound and outbound project license clearly

### CODE REVIEW

File: BestpracticesTemplate.md [hunk 1]
```diff
@@ -103,9 +103,9 @@ Ensure you adhere to the minimum content requirements for establishing README, C
 
 * README: description of project
 
-* CONTRIBUTING: contributing guidelines, how to get involved
+* CONTRIBUTING: contributing guidelines, how to get involved, and identifiction of license or agreement under which contributions are accepted
```
Comment (gregkh) [inline BestpracticesTemplate.md]: identifiction?

**Review by gregkh — CHANGES_REQUESTED** [review]
spell check and I'll be glad to merge it :)

**Comment by jlovejoy** [comment]
fixed now :)

**Review by gregkh — APPROVED** [review]
Thanks for making the change.

**Pull request merged**